### DISCUSSION TITLE
Is AVX completely disabled in KoboldCPP this days?

### DISCUSSION THREAD

**Discussion by ManOfFlash** [LostRuins/koboldcpp#358] [Q&A] [answered]
In some earlier versions i have output from KoboldCPP

> System Info: AVX **= 1** 

Now, in koboldcpp_nocuda1.37.1.exe it becomes **= 0**
My command string:

> koboldcpp.exe --noavx2 --noblas --threads 4 --launch %DBPATH%

How harm this cancelling?

**Comment by ManOfFlash** [comment]
I remove --noblas and AVX1 now = 1
But it crashes :(

**Reply by LostRuins** [reply, marked as answer]
Please try the latest version 1.38

**Reply by ManOfFlash** [reply]
Crashing go away now. Thanks!
